internal/compose/compose.go

```diff
@@ -340,7 +345,29 @@ func (p *Project) runDockerComposeCmd(opts dockerComposeOptions) error {
 	return cmd.Run()
 }
 
+func (p *Project) DockerComposeVersion() (string, error) {
+
+	const (
+		v1marker = "docker-compose version 1"
+	)
+	var b bytes.Buffer
+
+	args := []string{"version"}
```

Comment:
I think there is a switch `--short`, that can strip all unnecessary information and even use semver library to parse the value.

Reply:
Awesome thanks !!

Reply:
Unresolved